usecases/traverser/explorer_test.go

```diff
@@ -267,6 +267,68 @@ func Test_Explorer_GetClass(t *testing.T) {
 		})
 	})
 
+	t.Run("when the _certainty prop is set", func(t *testing.T) {
+		params := GetParams{
+			Kind:         kind.Thing,
+			Filters:      nil,
+			ClassName:    "BestClass",
+			Pagination:   &filters.Pagination{Limit: 100},
+			SearchVector: []float32{1.0, 2.0, 3.0},
+			Explore: &ExploreParams{
+				Values:    []string{"foobar"},
+				Limit:     100,
+				Certainty: 0,
+			},
+			UnderscoreProperties: UnderscoreProperties{
+				Certainty: true,
+			},
+		}
+
+		searchResults := []search.Result{
+			{
+				Kind: kind.Action,
+				ID:   "id2",
+				Schema: map[string]interface{}{
+					"age": 200,
+				},
+				Vector: []float32{0.5, 1.5, 0.0},
+			},
+		}
+
+		search := &fakeVectorSearcher{}
+		vectorizer := &fakeVectorizer{}
+		extender := &fakeExtender{}
+		log, _ := test.NewNullLogger()
+		projector := &fakeProjector{}
+		pathBuilder := &fakePathBuilder{}
+		explorer := NewExplorer(search, vectorizer, newFakeDistancer69(), log, extender, projector, pathBuilder)
+		expectedParamsToSearch := params
+		expectedParamsToSearch.SearchVector = []float32{1.0, 2.0, 3.0}
+		// expectedParamsToSearch.SearchVector = nil
+		search.
+			On("VectorClassSearch", expectedParamsToSearch).
+			Return(searchResults, nil)
+
+		res, err := explorer.GetClass(context.Background(), params)
+
+		t.Run("class search must be called with right params", func(t *testing.T) {
+			assert.Nil(t, err)
+			search.AssertExpectations(t)
+		})
+
+		t.Run("response must contain concepts", func(t *testing.T) {
+			require.Len(t, res, 1)
+
+			resMap := res[0].(map[string]interface{})
+			assert.Equal(t, 2, len(resMap))
+			assert.Contains(t, resMap, "age")
+			assert.Equal(t, 200, resMap["age"])
+			assert.Contains(t, resMap, "_certainty")
+			// Certainty is fixed to 0.5 in this mock
+			assert.InEpsilon(t, 0.31, resMap["_certainty"], 0.000001)
```

Comment:
Nice! I didn't know `InEpsilon` existed ;-)

Reply:
Oh I just see that the comment is still old it should be `fixed to 0.69` 0.5 is problematic because it does indicate if the distance or the certainty is returned since they are the same at that number.

Reply:
Good point, I didn't see that.